packages/appeals-service-api/src/schemas/lpa.spec.js

```diff
@@ -236,12 +236,12 @@ describe('LPA schema test', () => {
 
     describe('#loadData', () => {
       it('should load the data from the files', async () => {
-        const lpaExample = `OBJECTID,LPA19CD,LPA19NM,BNG_E,BNG_N,LONG,LAT,Shape__Area,Shape__Length
-1,E60000001,County Durham LPA,410381,532242,-1.8405,54.68513,2231526181.85083,315240.533401901
-390,W43000018,Cardiff LPA,315270,178887,-3.22212,51.50254,142287245.75211,79678.4715379399
-2,E60000002,Darlington LPA,428029,515648,-1.56835,54.53534,197475688.992432,107206.401677287
-331,N13000004,Belfast LPA,146465,529747,-5.92535,54.59853,137716630.472664,65908.6538260691
-343,S44000005,City of Edinburgh LPA,320193,669416,-3.27826,55.91119,263351803.906158,108121.329976751`;
+        const lpaExample = `OBJECTID,LPA19CD,LPA19NM,email,domain
+1,E60000001,County Durham LPA,,
+390,W43000018,Cardiff LPA,,
+2,E60000002,Darlington LPA,,
+331,N13000004,Belfast LPA,,
+343,S44000005,City of Edinburgh LPA,,`;
```

Comment:
worth adding extra tests for when `email` and / or `domain` are specified, e.g.

`68,E60000068,Bradford,appealsplanning@bradford.gov.uk,bradford.gov.uk`

?

Reply:
I'll sort this shortly

Reply:
i think this is where your documentation is Chris
https://pins-ds.atlassian.net/wiki/spaces/AAPDS/pages/457703525/DRAFT+Operational+Support+Manual

Reply:
I have moved this into a sub document of that Operational Support Manual page:

https://pins-ds.atlassian.net/wiki/spaces/AAPDS/pages/632487948/Eligible+Local+Planning+Departments